subprojects/docs/src/docs/userguide/migration/upgrading_version_7.adoc

```diff
@@ -171,14 +171,15 @@ These deprecated elements will be removed in a future version of Gradle.
 [[referencing_script_configure_method_from_container_configure_closure_deprecated]]
 ==== Implicitly referencing outer scope methods in some configuration blocks in Groovy DSL is now deprecated
 
-Prior to Gradle 7.6, Groovy scripts would permit accessing root project-level configure methods from
-within named container configure methods, as long as that container configure method would otherwise
-fail with a `MissingMethodException`. Consider the following snippets for examples of this behavior:
-
-Accessing the top-level `repositories` block from within the `configurations` block is permitted
-as long as the provided closure is otherwise an invalid configure closure for a Configuration.
-In this case, the `repositories` closure is executed as if it were called at the script-level, _and_
-an unconfigured `repositories` Configuration is created.
+Prior to Gradle 7.6, Groovy scripts permitted access to root project configure methods
+within named container configure methods that handle failures with a `MissingMethodException`.
```

Comment:
I'm thinking of a wording more like:
```suggestion
within named container configure methods that throw `MissingMethodException`s.
```
Using the word `throw` here is a bit more accurate